curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-numbers-in-a-range.english.md

```diff
@@ -7,9 +7,19 @@ challengeType: 5
 
 ## Description
 <section id='description'>
-We'll pass you an array of two numbers. Return the sum of those two numbers plus the sum of all the numbers between them.
-The lowest number will not always come first.
+We'll pass you an array of two numbers. Return the sum of those two numbers plus the sum of all the numbers between them. The lowest number will not always come first.
+
 Remember to use <a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code.
+ 
+### Example
```

Comment:
This heading is also github flavoured markdown.
I think changing it to simply: 
>Example:

Would be fine since other challenges follow the same style.